chart/openfaas/README.md

```diff
@@ -219,6 +219,23 @@ By default services will be exposed with following hostnames (can be changed, se
 
 If you require TLS/SSL then please make use of an IngressController. A full guide is provided to [enable TLS for the OpenFaaS Gateway using cert-manager and Let's Encrypt](https://docs.openfaas.com/reference/ssl/kubernetes-with-cert-manager/).
 
+### Istio mTLS
```

Comment:
(Should we mention an Istio version that it's tested with?)

Reply:
Done